server/configuration.go

```diff
@@ -91,6 +92,7 @@ func (p *Plugin) setConfiguration(configuration *configuration) {
 	}
 
 	p.configuration = configuration
+	p.zoomClient = zoom.NewClient(configuration.ZoomAPIURL, configuration.APIKey, configuration.APISecret)
```

Comment:
A bit of a nit pick, but I would like to see this line inside of `OnConfigurationChange` as it doesn't have to do with `setConfiguration`